static/app/stores/pageFiltersStore.tsx

```diff
@@ -74,9 +65,9 @@ const storeConfig: Reflux.StoreDefinition & Internals & PageFiltersStoreInterfac
 
   getState() {
     const isReady = this._hasInitialState;
-    const {selection} = this;
+    const {selection, organization} = this;
```

Comment:
It's likely we could lookup the organization in the OrganizationStore, but for now let's be safe and keep it here 🤔